doc-content/enterprise-only/weblogic/wls-install-start-proc.adoc

```diff
@@ -70,9 +70,11 @@ The startup script displays a series of messages, and finally displays a message
 http://<HOST>:<PORT>/console
 ----
 
-`<HOST>` is the system name or IP address of the host server.
+In this command, replace the following place holders:
 
-`<PORT>` is the address of the port on which the host server is listening for requests (7001 by default).
+* Replace `<HOST>` with the system name or IP address of the host server.
+
+* Replace `<PORT>` with the address of the port on which the host server is listening for requests (7001 by default).
```

Comment:
[Style] The port variable is a number rather than an address. Replace "on which".

s/the address of the port on which the host server is listening for requests (7001 by default)./the port number that the host server is listening on (7001 by default).